spark-on-dsx/json-array-object-to-db2-data-type.md

```diff
@@ -0,0 +1,65 @@
+# Transform JSON array or object into separate Db2 column data types
+
+The Python and Scala code below will demonstrate how to unpack a JSON array or object into
+separate Db2 columns.
+
+Both examples below will use this JSON document example:
+```
+{
+  "_id": "00000e46-ad8a-46d7-bd60-ebe03fc84d7f",
+  "_rev": "1-e2bb3326c19640eb830779b04b8019df",
+  "array": [
+    "Aberystwyth",
+    "Dorchester",
+    9
+  ],
+  "object": {
+    "id": "1",
```

Comment:
Probably best to name this field something else given the use of `_id` in Cloudant documents, it could get a bit confusing.
Also take care with this being a string number vs a JSON number.

Reply:
Changed key `id` to `number` with a JSON number type in 6f36e53.